src/index.js

```diff
@@ -14,43 +27,17 @@ export function createGovernor(initialState = {}, actions = {}) {
     );
   }
 
-  class HookActions {
-
-    constructor() {
-      for (let key in actions) {
-        if (key === 'dispatch' || key === 'state') {
-          throw new Error('Cannot name actions as "dispatch" or "state"');
-        }
```

Comment:
I don't think there is any problem with putting keys like `dispatch` on the actions.

Reply:
`this.dispatch` is reserved for the dispatch function. You'd lose an action named `dispatch`.